src/transport.cpp

```diff
@@ -853,6 +854,9 @@ namespace quicr {
         } else {
             auto pub_n_it = pub_ns_it->second.find(th.track_name_hash);
             if (pub_n_it == pub_ns_it->second.end()) {
+                track_handler->SetStatus(pub_ns_it->second.begin()->second->GetStatus());
+                SendAnnounce(conn_it->second, tfn.name_space);
```

Comment:
We now send duplicate announces, one for each publish track.  The reason is that we need the relay to be notified that there is a new publish track and a subscribe that may have errored last time should be resent now.